edx_proctoring/templates/emails/proctoring_attempt_status_email.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+
+Dear {{ student_name }},
+
+    This email is to notify that status of {{ exam_name }} for {{ course_name }} has changed to {{ status }} on {{ date }}.
+
+Thank you.
+
+
```

Comment:
@chrisndodge @griffresch What do you mean by "real text"? If the intention of these emails is simply to notify students that the status of their proctoring review has changed, it could be something like this: 

"This email is to let you know that the status of your proctoring review for the {NAME} in {COURSE} has changed to {STATUS}."

I'd suggest that if the course name/number appears in the subject, it doesn't need to be repeated in the body text. I did not include the date; I'm assuming that the actual change triggers these emails, so the change date = the email date. I'm not sure it's that valuable to students to explicitly be told a date, but I could be wrong.
Thoughts?